ister.py

```diff
@@ -1434,21 +1460,23 @@ def handle_options():
     group.add_argument("-F", "--fast-install", action="store_true",
                         default=False,
                         help="Move swupd state dir inside image for a faster install")
-    args = parser.parse_args()
+    args = parser.parse_args(sys_args)
     return args
 
 
 def main():
     """Start the installer
     """
     global LOG
-    args = handle_options()
+    args = handle_options(sys.argv[1:])
```

Comment:
So if we are not passing sys.argv[0] then we should probably set prog= in the ArgumentParser initialization (prog="ister" would make sense to me).

Reply:
updated